outlierdetect/outlierdetect.py

```diff
@@ -105,49 +105,54 @@ def compute_outlier_scores(self, frequencies):
                     (number of times aggregation unit reported value).
 
             Returns:
-                - dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
-                - dictionary mapping (aggregation unit) ->  return from _sum_frequencies: (a dictionary mapping (value) -> (number of times all aggregation units apart from agg_unit reported this value)).
+                dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
```

Comment:
Double check this? I think we added one return value in the tuple instead of removing?